File: mobile/src/main/java/com/db/williamchartdemo/MainActivity.java
package com.db.williamchartdemo;

import android.content.ActivityNotFoundException;
import android.content.Intent;
import android.net.Uri;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.Menu;
import android.view.MenuItem;


public class MainActivity extends AppCompatActivity {


	@Override
	protected void onCreate(Bundle savedInstanceState) {

		super.onCreate(savedInstanceState);
		setContentView(R.layout.main);
		getSupportFragmentManager().beginTransaction().add(R.id.container, new ChartsFragment()).commit();
	}


	@Override
	public boolean onCreateOptionsMenu(Menu menu) {

		getMenuInflater().inflate(R.menu.menu, menu);
		return super.onCreateOptionsMenu(menu);
	}


	@Override
	public boolean onOptionsItemSelected(MenuItem item) {

		switch (item.getItemId()) {
			case R.id.github:
				startActivity(new Intent("android.intent.action.VIEW",
						  Uri.parse("https://github.com/diogobernardino/WilliamChart")));
				return true;
			case R.id.play:
				try {
					startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse(
							  "market://details?id=" + this.getApplicationContext().getPackageName())));
				} catch (ActivityNotFoundException e) {
					startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse(
							  "http://play.google.com/store/apps/details?id=" +
										 "https://play.google.com/store/apps/details?id=com.db.williamchartdemo")));
				}
				return true;
			default:
				return super.onOptionsItemSelected(item);
		}
	}

}

File: mobile/src/main/java/com/db/williamchartdemo/MainActivity.kt
package com.db.williamchartdemo

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.google.android.material.bottomnavigation.BottomNavigationView
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    private val mOnNavigationItemSelectedListener = BottomNavigationView.OnNavigationItemSelectedListener { item ->
        when (item.itemId) {
            R.id.navigation_linechart -> {
                supportFragmentManager.beginTransaction().apply {
                    replace(R.id.contentFrame, LineChartFragment.newInstance())
                    commit()
                }
                return@OnNavigationItemSelectedListener true
            }
            R.id.navigation_barchart -> {
                supportFragmentManager.beginTransaction().apply {
                    replace(R.id.contentFrame, BarChartFragment.newInstance())
                    commit()
                }
                return@OnNavigationItemSelectedListener true
            }
        }
        false
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        navigation.setOnNavigationItemSelectedListener(mOnNavigationItemSelectedListener)
        navigation.selectedItemId = R.id.navigation_linechart
    }
}
